docs/tr/docs/python-types.md

```diff
@@ -210,60 +242,192 @@ Bir `dict` tanımlamak için virgülle ayrılmış iki parametre verebilirsiniz.
 
 İkinci parametre ise `dict` değerinin `value` değeri içindir:
 
-```Python hl_lines="1  4"
-{!../../../docs_src/python_types/tutorial008.py!}
-```
+=== "Python 3.9+"
+
+    ```Python hl_lines="1"
+    {!> ../../../docs_src/python_types/tutorial008_py39.py!}
+    ```
+
+=== "Python 3.6+"
+
+    ```Python hl_lines="1  4"
+    {!> ../../../docs_src/python_types/tutorial008.py!}
+    ```
 
 Bu şu anlama gelir:
 
 *  `prices` değişkeni `dict` tipindedir:
     *  `dict` değişkeninin `key` değeri  `str` tipindedir (herbir item'ın "name" değeri).
     *  `dict` değişkeninin `value` değeri `float` tipindedir (lherbir item'ın "price" değeri).
 
-#### `Optional`
+#### Union
+
+Bir değişkenin `int` veya `str` gibi **çeşitli türlerden** herhangi biri olabileceğini bildirebilirsiniz.
+
+Python 3.6 ve üzeri sürümlerde (Python 3.10 dahil) `typing`den `Union` türünü kullanabilir ve kabul edilecek olası türleri köşeli parantezlerin içine koyabilirsiniz.
+
+Python 3.10'da olası türleri dikey bir <abbr title='"bitsel veya operatör" olarak da adlandırılır, ancak bu anlam burada geçerli değildir'>çubukla (`|`)</abbr> (|) ayırabileceğiniz yeni bir **sözdizimi** de var.
+
+=== "Python 3.10+"
+
+    ```Python hl_lines="1"
+    {!> ../../../docs_src/python_types/tutorial008b_py310.py!}
+    ```
+
+=== "Python 3.6+"
 
-`Optional` bir değişkenin `str`gibi bir tipi olabileceğini ama isteğe bağlı olarak tipinin `None` olabileceğini belirtir:
+    ```Python hl_lines="1  4"
+    {!> ../../../docs_src/python_types/tutorial008b.py!}
+    ```
+
+Her iki durumda da bu, `item`in bir `int` veya `str` olabileceği anlamına gelir.
```

Comment:
``Her iki kullanımda `item`ın bir `int` veya `str` değer alabileceğini ifade eder.``

Lütfen ek kullanırken Türkçe okuyormuşuz gibi değilde İngilizce okuyormuşuz gibi hareket edelim. `FastAPI'nin` veya `item'in` kullanımı yanlıştır.